pyproject.toml

```diff
@@ -4,6 +4,7 @@ authors = [
   { name = "Matt Fisher", email = "mfisher87@gmail.com" },
   { name = "Rushiraj Nenuji", email = "nenuji@nceas.ucsb.edu" },
   { name = "Trey Stafford", email = "trey.stafford@colorado.edu" },
+  { name = "Robyn Marowitz", email = "robyn.marowitz@colorado.edu" },
```

Comment:
Nice, good catch! Maybe a note about adding oneself to the authors list in the contributing doc would be good.